UPGRADING.md

```diff
@@ -5,6 +5,15 @@ This document will be updated to list important BC breaks and behavioral changes
 
 ### upgrading to 2.0.0
 
+ * The SensioFrameworkExtraBundle `@View` annotation must be enabled to use the `ViewResponseListener`:
```

Comment:
Agreed, we use the same wording in the `ConfigurationCheckPass` exception message. So it makes sense to use that here too.